src/main/scala/org/camunda/feel/impl/interpreter/FeelInterpreter.scala

```diff
@@ -870,11 +782,7 @@ class FeelInterpreter {
     }
 
     function.invoke(paramList) match {
-      case ValError(failure) => {
-        // TODO (saig0): customize error handling (#260)
-        logger.warn(s"Failed to invoke function: $failure")
-        ValNull
-      }
+      case e: ValError => e
```

Comment:
🤔 This might be the answer to the behavioural change I mentioned earlier

Reply:
Yes. Now, the error is propagated to the caller of `invokeFunction()` to report an error that contains also the name of the function.